thrust/thrust/type_traits/is_contiguous_iterator.h

```diff
@@ -31,13 +31,9 @@
 #elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_MSVC)
 #  pragma system_header
 #endif // no system header
-#include <thrust/detail/raw_pointer_cast.h>
-#include <thrust/detail/type_traits.h>
-#include <thrust/detail/type_traits/pointer_traits.h>
+#include <thrust/detail/type_traits/is_thrust_pointer.h>
 
-#include <iterator>
-#include <type_traits>
-#include <utility>
+#include <cuda/std/iterator> // Needed for __gnu_cxx::__normal_iterator
```

Comment:
No change requested, but all of this needs to go an be replaced by `::cuda::std::contiguous_iterator`